docs/spec/manifest-v2-2.md

```diff
@@ -0,0 +1,109 @@
+# Distribution Content Manifests
+
+A Content Manifest is a simple JSON file which contains general fields that
+describe an object and its dependencies. The goal is for these manifests to
+describe a reference to any application data, optional metadata or labels, and
+to reference any dependent objects in a content-addressable and verifiable way.
+These manifests can also be stored in a repository and referenced by digest.
+
+## Field Descriptions
+
+- **`schemaVersion`** *int*
+
+    Specifies the schema version of the manifest as an integer. This document 
+    describes version `2` only.
+
+- **`target`** *object*
+
+    The target field may describe *any* object stored in a repository, allowing
```

Comment:
@jlhawn 

> If your container image depended on another base image, then that base image would be in another repository and your image would refer to the base image by its full name and tag/digest reference.

I'm guessing the Docker image manifest format would need to be modified to support this?